### PR TITLE
Issue/5 endpoint for static data

### PR DESCRIPTION
**Pull request by emilhaugberg**
Still a bit hacky, but it does the job. 

@enirolf and I have:

- added endpoint for adding stops to the database
- added endpoint for adding transport lines to the database

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -8,6 +8,37 @@
 
 server = PromInsertServer()
 
+def make_transport_line(operator_id, line_id, transport_type_id, line_obj):
```
Comment (emilhaugberg) [inline app/main.py:11]: We probably want to move these kinds of functions elsewhere

File: app/main.py [hunk 2]
```diff
@@ -159,7 +197,71 @@ def get_stops():
...
+    internal_id = request.args.get("internal_id", default="%", type=str).split(",")
+
+    sql = MysqlConnector()
+
+    line_id = sql.getId("transport_lines", {"internal_id": internal_id[0]})
+    stop_query = build_query("transport_lines_stops", ["id"], {"transport_line_id": [line_id]}) + "ORDER BY order_number"
+    stop_ids = list(sql.execQuery(stop_query))
+
+    #TODO Error handling! Not every stop_id has a corresponding stop for some reason.
+    stops = [sql.execQuery(build_query("stops", ["stop_code"], {"id": [stop_id[0]]}))[0][0] for stop_id in stop_ids]
```
Comment (enirolf) [inline app/main.py:254]: This is still very hacky and does require some rewriting/error handling, but it does the job for now and we need this for analytics.

Comment (emilhaugberg) [reply app/main.py:254]: We'll fix it after the demo this week 👍

**Comment by emilhaugberg** [comment]
Made obsolete by #7

**Pull request merged**